### DISCUSSION TITLE
Does spfx supports react-adaptive cards

### DISCUSSION THREAD

**Discussion by cbathis** [pnp/sp-dev-fx-webparts#1717] [General]
We are trying to upgrade adaptive card from 1.0 to 1.3 but it will require bot framework 4.11 which is dependent on node 14.x

As spfx webpart supported on node 8.x and 10.x only, how this solution (adaptive 1.3) will support on spfx webpart with ?

Please assist.

**Comment by hugoabernier** [comment]
I know that the SPFx is planning for Node 14. support for upcoming SPFx versions, but I am not sure whether it'll be for SPFx 1.12 or later.

According to Vesa Juvonen on the SPFx Special Interest community call last week, SPFx 1.12 is a matter of weeks now, not months.

I'd suggest holding off and see?

I'll try to flag this conversation back as soon as I can confirm that Node 14 support is available.

**Reply by cbathis** [reply]
Thanks for the update

**Reply by hugoabernier** [reply]
SPFx 1.12, released on March 15, does not support Node v14.x yet -- only Node v12.x for now. 

Based on the updates on the latest community call, v14.x support has been pushed back to SPFx 1.13
